frontend/src/components/table/TableRowTitleDescription.tsx

```diff
@@ -19,7 +19,7 @@ type TableRowTitleDescriptionProps = {
 
 const TableRowTitleDescription: React.FC<TableRowTitleDescriptionProps> = ({
   title,
-  boldTitle = true,
+  boldTitle = false,
```

Comment:
If we are changing the default to `false` we can probably remove the prop all together and update any instances that explicitly set `false` to remove the setting. Are there any that explicitly want it to be `true`?